main.rb

```diff
@@ -3,61 +3,71 @@
 require './rental'
 require './student'
 require './teacher'
-def add_student
-  puts 'Name'
-  name = gets.chomp
-  puts 'Age'
-  age = gets.chomp
-  puts 'Has parent permission (Y/N)'
-  has_permission = gets.chomp
-  has_permission = has_permission == 'y'
-  student = Student.new(age, '', name, parent_permission: has_permission)
-  puts 'Student created successfully'
-  student
+class AddStudent
+  def add_student
+    puts 'Name'
+    name = gets.chomp
+    puts 'Age'
+    age = gets.chomp
+    puts 'Has parent permission (Y/N)'
+    has_permission = gets.chomp
+    has_permission = has_permission == 'y'
+    student = Student.new(age, '', name, parent_permission: has_permission)
+    puts 'Student created successfully'
+    student
+  end
 end
 
-def add_teacher
-  puts 'Name'
-  name = gets.chomp
-  puts 'Age'
-  age = gets.chomp
-  puts 'Specialization'
-  specialization = gets.chomp
-  teacher = Teacher.new(age, specialization, name, parent_permission: nil)
-  puts 'Teacher created successfully'
-  teacher
+class AddTeacher
+  def add_teacher
+    puts 'Name'
+    name = gets.chomp
+    puts 'Age'
+    age = gets.chomp
+    puts 'Specialization'
+    specialization = gets.chomp
+    teacher = Teacher.new(age, specialization, name, parent_permission: nil)
+    puts 'Teacher created successfully'
+    teacher
+  end
```

Comment:
- [ ] Let's try to refactor every user input in the manager classes to a single place that will handle only that responsibility. That way we will have a class that is in charge of all the inputs.

[OPTIONAL] I would also recommend having classes in separate files so that they're easier to navigate and read.